src/Simplex/Chat.hs

```diff
@@ -614,6 +630,26 @@ processChatCommand = \case
     ChatConfig {defaultServers = InitialAgentServers {smp = defaultSMPServers}} <- asks config
     withAgent $ \a -> setSMPServers a (fromMaybe defaultSMPServers (nonEmpty smpServers))
     pure CRCmdOk
+  APISetChatItemTTL newTTL -> withUser $ \user -> withChatLock $ do
```

Comment:
withChatLock - is it needed here?

Reply:
should we maybe require that chat is stopped before allowing to change this settings?